bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleEngine.java

```diff
@@ -1015,4 +1023,14 @@ public void templateUpdated(Collection<Template> templates) {
 
     }
 
+    public void setEventPublisher(EventPublisher ep) {
+        this.eventPublisher = ep;
```

Comment:
Didn´t we introduced some kind of callback, so that we don´t need the EventPublisher in the RuleEngine?